scss/helpers/_colored-links.scss

```diff
@@ -1,11 +1,11 @@
 @each $color, $value in $theme-colors {
   .link-#{$color} {
-    color: $value;
+    --#{$variable-prefix}link-color: #{$value};
 
     @if $link-shade-percentage != 0 {
       &:hover,
       &:focus {
-        color: if(color-contrast($value) == $color-contrast-light, shade-color($value, $link-shade-percentage), tint-color($value, $link-shade-percentage));
+        --#{$variable-prefix}link-hover-color: #{if(color-contrast($value) == $color-contrast-light, shade-color($value, $link-shade-percentage), tint-color($value, $link-shade-percentage))};
       }
```

Comment:
This should probably not be set on `:hover` nor `:focus` since a non-link element wouldn't have focus.

Drop the `:hover, :focus` selector completely :)

Reply:
:D now I understand, dropped